### PR TITLE
Added $dateFormat to BrowserContext::__construct()

### PR DESCRIPTION
**Pull request by zhangwei**
Hi

It will be great if anyone can change date format.

### CODE REVIEW

**Comment by sanpii** [comment]
Good idea! Can you add a test?

**Comment by zhangwei** [comment]
@sanpii, I  have already added some tests.

**Comment by sanpii** [comment]
Sorry, I have not been clear. You added a new step but I would like a new scenario for testing `$dateFormat` modification.

File: src/Context/BrowserContext.php [hunk 1]
```diff
@@ -10,11 +10,12 @@
 class BrowserContext extends BaseContext
 {
     private $timeout;
-    private $dateFormat = 'dmYHi';
+    private $dateFormat;
 
-    public function __construct($timeout = 1)
+    public function __construct($timeout = 1, $dateFormat = 'Y-m-d')
```
Comment (sanpii) [inline src/Context/BrowserContext.php]: The default `$dateFormat` should be `dmYHi` to prevent incompatibilities.

Comment (zhangwei) [inline src/Context/BrowserContext.php]: Yes, I have changed src as above.

**Comment by zhangwei** [comment]
@sanpii, Thank you for your feedback.

I saw there was some scenarios already. So should I change the behat.yml.dist to use it ? ( 1f668e6 )

**Pull request closed without merging**